artman/cli/main.py

```diff
@@ -113,6 +111,14 @@ def parse_args(*args):
              '/path/to/config.yaml:config_section_A|config_section_B. '
              'This is an advanced feature; for most cases, use `--api`.',
     )
+    api.add_argument('--batch',
+        action='store_true',
+        help='Enables batch mode, which will generate all APIs for the '
```

Comment:
Did we really have this functionality? I find that surprising.

This is also something that it might be easier to just use a shell loop:

```bash
for LANGUAGE in ( python java nodejs ruby php go ); do
  artman --api=pubsub --language=$LANGUAGE --publish=github
done
```